issues.go

```diff
@@ -116,6 +116,7 @@ type Issue struct {
 	MergeRequestCount    int              `json:"merge_requests_count"`
 	EpicIssueID          int              `json:"epic_issue_id"`
 	Epic                 *Epic            `json:"epic"`
+	MovedToID            int              `json:"moved_to_id"`
```

Comment:
Please place this one between `CreatedAt` and `Labels` so it follows the ordering in the docs.